{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/core/admin.py

```diff
@@ -57,7 +65,7 @@ class CustomUserAdmin(UserAdmin):
     ordering = []
 
     def permissions(self, obj):
-        return ", ".join([g.name for g in obj.groups.all()])
+        return [g.name for g in obj.groups.all()]
```

Comment:
@oudeismetis - I trust that you checked that this works? This is changing the return type from `str` to `list[str]`.

Reply:
correct. I pulled this from a project where I made the change.
The join isn't needed and happens for you

Reply:
Just re-tested on a locally generated my_project.
![image](https://github.com/user-attachments/assets/e4bd3af3-8501-4c20-8026-482d12984c23)

Reply:
Great! This is good to go then.